src/provider/provider_level_zero.c

```diff
@@ -267,6 +297,15 @@ static void init_ze_global_state(void) {
         return;
     }
     ze_lib_handle = lib_handle;
+
+    umf_result_t result = ze_init_drivers();
```

Comment:
Hmmm, I am thinking about who is responsible for calling `zeInitDrivers`.
I suppose that the client who uses UMF and Level Zero is responsible for that. At the moment, the client calls UMF to create Level Zero it should already create the context and get the device, so the driver should be initialized.

Reply:
I agree with @vinser52

Reply:
As discussed, UMF needs to call init function, even if the client called it already.